mockbee-test/ecom-store/src/productUtilities/filter-by-stock.js

```diff
@@ -0,0 +1,18 @@
+const getProductsByStock = (products, includeOutOfStock) => {
+    let filtertedList = [];
+    let newList = [];
+    if (includeOutOfStock) {
+      newList = products.filter(
+        ({ outOfStock }) => outOfStock !== includeOutOfStock
+      );
+      filtertedList.push(...newList);
+    }
+    newList = products.filter(
+      ({ outOfStock }) => outOfStock === includeOutOfStock
+    );
+    filtertedList.push(...newList);
+    return filtertedList;
+  };
+  
+  export { getProductsByStock };
```

Comment:
Add a doc comment for what this function is doing

Reply:
sure will add a doc comment